### DISCUSSION TITLE
spritepack gifts are not recognized

### DISCUSSION THREAD

**Discussion by wowm03** [Monika-After-Story/MonikaModDev#9826] [QA - tech support + general] [answered]
my friend's Monika has been failing to read the gift **formal dress** from the officially released spritepacks.

this also applies to all other spritepacks. Christmas passed and there are no presents under the Christmas tree.
![image](https://user-images.githubusercontent.com/90627231/210074492-4c417f9c-e6d9-49e3-965f-f96c2c62a1a7.png)

however, she can still be gifted things like roses, fudge, and cupcakes.
![image](https://user-images.githubusercontent.com/90627231/210074497-c70a50fe-6e9d-4686-ad00-31285c6ec738.png)

here is the persistent
[persistent.zip](https://github.com/Monika-After-Story/MonikaModDev/files/10324880/persistent.zip)

thank you.

**Comment by Booplicate** [comment, marked as answer]
> formal dress from the officially released spritepacks

We don't have "formal dress," that's something unofficial.
For installing spritepacks follow [this video](https://www.youtube.com/watch?v=bjJ0U8l1m8c). And [this guide](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation).
